source/samples/pubsub/PubSubFeature.cpp

```diff
@@ -152,7 +152,7 @@ int PubSubFeature::getPublishFileData(aws_byte_buf *buf)
 void PubSubFeature::publishFileData()
 {
     ByteBuf payload;
-    if (pubFile == "")
+    if (FileUtils::GetFileSize(pubFile) == 0)
```

Comment:
Are we adding default message in publish file while creating it? If not then the issue will not solve the problem of our new DC users who has not manually update the file data.